rs-matter-macros-impl/src/lib.rs

```diff
@@ -3,6 +3,8 @@ use proc_macro2::{Ident, Literal, Span, TokenStream};
 use quote::quote;
 use rs_matter_data_model::Cluster;
 
+pub const IDL: &str = include_str!("idl/controller-clusters.matter");
```

Comment:
I would name these CSA_STANDARD_CLUSTERS_IDL or similar and move into rs-matter-macros (as it is unused in impl).

Reply:
Renamed. Shall we leave it in the `rs-matter-macros-impl` crate nevertheless? I would imagine, we might need it in future for some tests?

Reply:
For tests I was thinking to have separate files that showcase specific behavior and are not subject to change. I am concerned that any official cluster may have stuff added to it in time and that would alter expected outputs.

The only test we may want is "we can parse the entire official cluster list" however that will happen anyway as part of our main macro usage.